test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenManagementSpecs.java

```diff
@@ -127,7 +129,14 @@ public HapiApiSpec wipeAccountFailureCasesWork() {
 						tokenCreate(wipeableToken)
 								.name(salted("name"))
 								.treasury(TOKEN_TREASURY)
-								.wipeKey("wipeKey")
+								.wipeKey("wipeKey"),
+						tokenCreate(anotherWipeableToken)
+								.name(salted("name"))
```

Comment:
You are safe to remove the `.name(salted(..))` property. In the `tokenCreate` it is added automatically